ucsb-cs156-f24/team02-f24-10
# Implemented create page for UCSBDiningCommonsMenuItem

Closes #13 

To test the page:
1. Either run the frontend locally or use my dev dokku deploy here: https://team02-dev-justinwu13.dokku-10.cs.ucsb.edu/
2. Log in, then navigate to the `/UCSBDiningCommonsMenuItem` index page by clicking on UCSB Dining Commons Menu Item in the navigation bar
3. Click the Create button to get to the create page, should look like this: <img width="1321" alt="Screenshot 2024-11-03 at 1 42 48 PM" src="https://github.com/user-attachments/assets/e20e5313-7b40-4d21-9ca6-5fdde5ef5174">
4. Fill out the form. There are no restrictions on the contents of each field except that they cannot be empty. A successful addition will cause a popup like the following: <img width="1129" alt="Screenshot 2024-11-03 at 1 47 36 PM" src="https://github.com/user-attachments/assets/fc2723b8-cdcb-4b60-be31-344612c44bd0">
5. Use the Swagger `/api/ucsbdiningcommonsmenuitems/all` endpoint to verify that your item was successfully created. <img width="1129" alt="Screenshot 2024-11-03 at 1 48 19 PM" src="https://github.com/user-attachments/assets/075951d8-85b0-4ae9-8c1d-7c321e3fa5ba">

for anyone reviewing the code, there's a little inconsistency where my API endpoints use "ucsbdiningcommonsmenuitems" while in other instances I use the singular version. might fix that in the future but it seems to work fine for now

storybook: https://6722c06126c430949f67f157-yvxdivxfsx.chromatic.com
